show/interfaces/__init__.py

```diff
@@ -319,6 +319,52 @@ def expected(db, interfacename):
 
     click.echo(tabulate(body, header))
 
+# 'mpls' subcommand ("show interfaces mpls")
+@interfaces.command()
+@click.argument('interfacename', required=False)
+@multi_asic_util.multi_asic_click_options
+def mpls(interfacename, namespace, display):
+    """Show Interface MPLS status"""
+
+    ctx = click.get_current_context()
+
+    appl_db = SonicV2Connector()
+    appl_db.connect(appl_db.APPL_DB)
+
+    if interfacename is not None:
+        interfacename = try_convert_interfacename_from_alias(ctx, interfacename)
+
+    # Fetching data from appl_db for intfs
+    keys = appl_db.keys(appl_db.APPL_DB, "INTF_TABLE:*")
+    intfs_data = {}
+    for key in keys if keys else []:
+        tokens = key.split(":")
+        # Skip INTF_TABLE entries with address information
+        if len(tokens) != 2:
+            continue
+
+        if (interfacename is not None) and (interfacename != tokens[1]):
+            continue
+
+        mpls = appl_db.get(appl_db.APPL_DB, key, 'mpls')
+        if mpls == '':
+            intfs_data.update({tokens[1]: 'disable'})
+        else:
+            intfs_data.update({tokens[1]: mpls})
+
+    header = ['Interface', 'MPLS State']
+    body = []
+
+    # Output name and alias for all interfaces
+    for intf_name in natsorted(list(intfs_data.keys())):
+        if clicommon.get_interface_naming_mode() == "alias":
+            alias = clicommon.InterfaceAliasConverter().name_to_alias(intf_name)
+            body.append([alias, intfs_data[intf_name]])
+        else:
+            body.append([intf_name, intfs_data[intf_name]])
+
```

Comment:
This is optional. One suggestion is provide an option for 'json' output (eg. show platform summary). This will be useful in unit testing.  
```
{
   interface : {
         "Ethernet0" : "enable",
         "Ethernet4" : "disable",
   }
}
```